clang/include/clang/Lex/PPCallbacks.h

```diff
@@ -22,11 +22,12 @@
 #include "llvm/ADT/StringRef.h"
 
 namespace clang {
-  class Token;
-  class IdentifierInfo;
-  class MacroDefinition;
-  class MacroDirective;
-  class MacroArgs;
+class Token;
+class IdentifierInfo;
+class MacroDefinition;
+class MacroDirective;
+class MacroArgs;
+struct LexEmbedParametersResult;
```

Comment:
WS only change

Reply:
Yeah, this was because clang-format is picky. I added `struct LexEmbedParametersResult;` and so it wanted to indent the entire block when I formatted the patch.

I'll land as an NFC and rebase on top of it.